astro/src/content/docs/release-notes/index.mdx

```diff
@@ -142,6 +143,10 @@ Looking for release notes older than 1.23.0? Look in the [release notes archive]
   We plan to stop publishing docker images for these architectures at the end of 2024. If you are actively using any of these architectures, please let us know how this could affect you by contacting support or reaching out to sales.
 </DeprecationWarning>
 
+<ThemeUpdateWarning>
+  See [theme upgrade notes](/docs/customize/look-and-feel/upgrade#version-1520) for details on updating an existing advanced theme to use the new date picker with enhanced styling and mobile support.
```

Comment:
When this renders, it looks like `theme upgrade notes` should match the case of the other link in the call out which is `Theme Upgrades`. 

My preference would be that the `ThemeUpdateWarning` always contains links to the theme release notes and theme upgrade. 

And then the copy added in the `ThemeUpdateWarning` is related to whatever changed. 

Example component:

```
---
import Aside from 'src/components/Aside.astro';

export interface Props {
  version: string;
}

const { version } = Astro.props as Props;

var fragment = "";
if (version) {
  fragment = "#version-" + version.replace(/\./g, '');
}
---

<Aside type="theme" title="Theme update">
<p>This version of FusionAuth includes changes to the default FusionAuth theme. If you have created custom themes in previous versions of FusionAuth, <strong>you will need to incorporate these changes into your custom themes</strong>.</p>

<slot/>

<p>
  See <a href=`/docs/customize/look-and-feel/upgrade${fragment}`>Theme Upgrade Notes</a> for details on updating an existing advanced themes specific for this release. See <a href="/docs/customize/look-and-feel/advanced-theme-editor#upgrading">Theme Upgrades</a> for additional information about the upgrade process.
</p>
</Aside>

```

Example Usage:

```
<ThemeUpdateWarning version="1.52.0"/>

<ThemeUpdateWarning version="1.50.0">
  The new consent prompt themed page requires the `scopeConsentField` macro and `resolveScopeMessaging` function to be defined in the *Helpers* template in order to render scope consent form fields. These *must* be added to a custom theme in order for it to function.
</ThemeUpdateWarning>
```

Reply:
> it looks like theme upgrade notes should match the case of the other link in the call out which is Theme Upgrades

Saw that, too. The other thought I had was maybe we should make an effort for the link text to match the title of the referenced page?

Reply:
could also wrap the slot in 
```html
<div class="prose-a:capitalize">
  <slot/>
</div>
```
 if we want to enforce consistency there, but I wouldn't want to do that globally

Reply:
the like the `version` suggestion a lot. 

I initially thought that `fragment = "#version-" + version.replace(/\./g, '');` should go into the `/tools/string` group but then I realized that 

`/docs/release-notes#version-1-53-1`

and

`/docs/customize/look-and-feel/upgrade#version-1520`

don't match.

And just to make you crazy there is also `/direct-download#1_53_0` ☹️ 

So maybe we don't do that here but we should standardize our fragment links

Reply:
My preference is to avoid using something like `prose-a:capitalize` whenever possible because it is then not obvious to the user and it encourages incorrect usage IMO. 

> So maybe we don't do that here but we should standardize our fragment links

It would be nice to standardize for sure if possible.

Reply:
I think I got all this incorporated.